libs/partners/pinecone/pyproject.toml

```diff
@@ -20,7 +20,9 @@ disallow_untyped_defs = "True"
 [tool.poetry.dependencies]
 python = ">=3.8.1,<3.13"
 langchain-core = ">=0.1.52,<0.3"
-pinecone-client = ">=3.2.2,<5"
+pinecone-client = "^5.0.0"
+aiohttp = "^3.9.5"
```

Comment:
from pypi it looks like inference might be exposed by sdk - can we use that instead of separate aiohttp install?